jupyter_core/utils/__init__.py

```diff
@@ -182,7 +177,17 @@ def ensure_event_loop(prefer_selector_loop: bool = False) -> asyncio.AbstractEve
         loop = asyncio.get_running_loop()
     except RuntimeError:
         if sys.platform == "win32" and prefer_selector_loop:
-            loop = asyncio.WindowsSelectorEventLoopPolicy().new_event_loop()
+            if (3, 14) <= sys.version_info < (3, 15):
+                # ignore deprecation only for 3.14 and revisit later.
+                with warnings.catch_warnings():
+                    warnings.filterwarnings(
+                        "ignore",
+                        DeprecationWarning,
```

Comment:
```suggestion
                        category=DeprecationWarning,
```

This is causing `TypeError` in downstream package `jupyterlab_server` when testing on Python 3.14 on Windows.
CC: @Carreau

Reply:
Opened https://github.com/jupyter/jupyter_core/pull/449